src/test/utils/testingHelper.ts

```diff
@@ -16,13 +16,14 @@ export type TestingRunParams = {
 
 type TestHelperParams = {
   testingRun: (testingRunParams: TestingRunParams) => void;
-} & { network: Network };
+} & { network: Network; onFork: boolean };
 
 export const testingHelper = ({
   network,
+  onFork,
```

Comment:
```suggestion
  onFork = true,
```
and in here, we can make it true to be the default if no value has been passed down and it would be less of a breaking change.